src/main/kotlin/dev/bright/vb6serializer/BinaryDecoder.kt

```diff
@@ -165,6 +165,15 @@ internal open class BinaryDecoder(
         )
     }
 
+    private fun decodeStringWithoutLength(): String {
```

Comment:
Again, I'm not really a fan of the `withoutLength` naming. What does it mean to have a string without length?  Here, we'll not be decoding string without length. It's rather that the length of the string is encoded in the bytes we read from. We don't have a constant length but rather a variable length.

Reply:
Fixed. Sorry that I've missed it